### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by revett**
### Problem

Nothing here...

### Solution

Add glue

### Notes

- None

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,4 @@
+FROM java:7
+MAINTAINER Charlie Revett <charlie@vidsy.co>
+
+RUN apt-get -y update
```
Comment (stevenjack) [inline Dockerfile:4]: You not want to add and run dynamodb jar?

Comment (revett) [reply Dockerfile:4]: Just needed something to build. Going to write full Dockerfile in next PR

Comment (stevenjack) [reply Dockerfile:4]: 👌

File: README.md [hunk 2]
```diff
@@ -1,2 +1,13 @@
...
+  (Docker) Container that runs AWS local DynamoDB.
+</p>
+
+## Usage
+
+```
+TODO
+```
+
+> Based off [spurious-io/dynamodb](https://github.com/spurious-io/dynamodb).
```
Comment (stevenjack) [inline README.md:13]: 😍

**Review by stevenjack — APPROVED** [review]

**Pull request merged**